### DISCUSSION TITLE
Sports teams?

### DISCUSSION THREAD

**Discussion by matthewmayer** [faker-js/faker#1585] [General]
It looks like some locales have data for sports teams in the format place+creatures e.g. "California Bears"

https://github.com/faker-js/faker/blob/next/src/locales/en/team/creature.ts

However there's no actual API for this.

Either the data could be removed if unused, or a new API could be added?

cf: https://github.com/faker-js/faker/discussions/1297

**Comment by ST-DDT** [comment]
We haven't decided yet, what to do with these data.
Some data we will most likely just shuffle around, other data will be removed.
However we first have to create some tooling to detect non-referenced files: https://github.com/faker-js/faker/issues/1347

**Comment by Shinigami92** [comment]
To add some context: these were always indirectly reachable data via `faker.helpers.fake('...')`
That makes it sadly a bit like a breaking change when we suddenly remove it

**Comment by matthewmayer** [comment]
I'm curious where they originally came from 😀

**Reply by ST-DDT** [reply]
They were just added at some point:
https://github.com/faker-js/faker/blob/0048b9f6cfd69378ca3ba2f588444e7bb8095d23/lib/locales/en.js#L6467

**Reply by matthewmayer** [reply]
FWIW I think it would be a fairly fun addition. 

The creatures should probably be capitalised i.e. California Dolphins not California dolphins as present. 

And might want to allow different patterns for localisation as the Place Creatures pattern is quite Americanised. Eg for en_GB could have X United, X City, X Wanderers etc.
